includes/editor-templates/panel-elements.php

```diff
@@ -37,6 +37,16 @@
 
 <script type="text/template" id="tmpl-elementor-element-library-element">
 	<div class="elementor-element">
+		<#
+			var active = obj.categories.includes( 'favorites' );
+			var properties = {
+				class: active ? 'active' : '',
+				icon: active ? 'eicon-heart' : 'eicon-heart-o',
+			};
```

Comment:
It's better to use ES6 syntax. Use `const` instead of `var`

Reply:
> It's better to use ES6 syntax. Use `const` instead of `var`

That was my first approach. But then I saw other views in this section and apparently most of them using var. Good to know that there's a movement to change those conventions. Will do.

Reply:
As a rule of thumb, avoid any non-ES6 code when possible (e.g. JQ, underscore.js, `require()`, etc.).
Don't mind the existing code, and even refactor JQ to ES6 when needed 😄